docs/best-practices.md

```diff
@@ -3,32 +3,8 @@ Best Practices
 
 This page is a placeholder for best practices we recommend you to review while implementing Vue InstantSearch.
 
-
-## Custom Component
```

Comment:
What do you think @vvo ? I have no strong opinion here. I would tend to go the SE suggestion ;)